example/samples/install_coreos_payload_full.json

```diff
@@ -0,0 +1,43 @@
+{
+    "options": {
+        "defaults": {
+            "hostname": "Coreos-node",
+            "installDisk": "/dev/sda",
+            "comport": "ttys0",
+            "repo": "http://172.31.128.1:9080/coreos/amd64-usr",
+            "rootPassword": "NotUsed",
+            "coreSshKeys": [ "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDJQ631/sw3D40h/6JfA+PFVy5Ofz6eu7caxbv0zdw4fTJsrFcOliHZTEtAvqx7Oa8gqSC6d7v61M0croQxtt1DGUcH2G4yhfdZOlK4pr5McwtX0T/APACdAr1HtP7Bt7u43mgHpUG4bHGc+NoY7cWCISkxl4apkYWbvcoJy/5bQn0uRgLuHUNXxK/XuLT5vG76xxY+1xRa5+OIoJ6l78nglNGrj2V+jH3+9yZxI43S9I3NOCl4BvX5Cp3CFMHyt80gk2yM1BJpQZZ4GHewkI/XOIFPU3rR5+toEYXHz7kzykZsqt1PtbaTwG3TX9GJI4C7aWyH9H+9Bt76vH/pLBIn rackhd@rackhd-demo"
+            ],
+            "users": [
+                {
+                    "name": "rackhd1",
+                    "password": "rackhd1",
```

Comment:
Seems using 123456 as password is more popular on other examples: Would be nicer to keeping consistency. 
https://github.com/RackHD/RackHD/blob/master/example/samples/install_esx_payload_full.json#L12
https://github.com/RackHD/RackHD/blob/master/example/samples/install_rhel_payload_full.json#L12